src/features/styled-component.ts

```diff
@@ -1,7 +1,7 @@
 import { NodePath } from '@babel/core';
 import generate from '@babel/generator';
 import * as t from '@babel/types';
-import get from 'lodash/get';
+import get from 'lodash.get';
```

Comment:
I really dislike these little packages. they may be smaller but it usually balloons the dep tree and number of transitive dependencies significantly. What is the actual cost saves between all of them over installing lodash. TBH I don't understand this sort of optimization, even on slow networks installing 70kb of text is not expensive, in the context of "putting files on disk to do development", I would imagine (unscientifically) having more limited deps, thereby reducing the amount of network hops and logic required by bundlers to install would be faster overall.

Reply:
`lodash` is 1.3 MB vs 23 KB for `lodash.get`

For instance, I don't have `lodash` in subdependecies of my projects. Adding extra MB will slow down CI by a few seconds.

Reply:
@ai can we replace lodash by native ECMAScript features?